tests/app/phase-tester-response/issue-pending/issue-pending.component.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { IssuePendingComponent } from '../../../../src/app/phase-tester-response/issue-pending/issue-pending.component';
+import { Issue, STATUS } from '../../../../src/app/core/models/issue.model';
+import { ISSUE_WITH_EMPTY_DESCRIPTION } from '../../../constants/githubissue.constants';
+import { Team } from '../../../../src/app/core/models/team.model';
+import { IssueComment } from '../../../../src/app/core/models/comment.model';
+import { EMPTY_TEAM_RESPONSE, PENDING_TUTOR_MODERATION } from '../../../constants/githubcomment.constants';
+import { TutorModerationTodoTemplate } from '../../../../src/app/core/models/templates/tutor-moderation-todo-template.model';
+
+describe('IssuePendingComponent', () => {
+    describe('.ngOnInit()', () => {
+        const dummyTeam: Team = new Team({
```

Comment:
Just something to discuss amongst the team! Currently the mock `dummyTeam` is defined curently in quite a few specs files..

It would be good to raise an issue to move these mocks into a constants file, else there are multiple sources of truth and it can be quite confusing!

Reply:
Yeah sounds good 👍

Reply:
Sure, I will raise an issue for this!